libs/seiscomp/gui/datamodel/eventlistview.cpp

```diff
@@ -3176,6 +3197,23 @@ void EventListView::readFromDatabase(const Filter &filter) {
 		}
 		it.close();
 
+		// fetch comments to be able to later mark published focal mechanisms
+		for (size_t i=0; i<ep.focalMechanismCount(); i++) {
+			_reader->loadComments(ep.focalMechanism(i));
+		}
+
+		/*
```

Comment:
Why choosing the slow version? Running `focalMechanismCount` additional single queries will slow down the overall event loading especially with remote databases.